docs/smart-contracts/data-types/primitive-data-types.md

```diff
@@ -15,6 +15,7 @@ The high-level languages may implement these data types slightly differently, bu
 - [Booleans](#booleans)
```

Comment:
(sorry I cannot comment above this line)

Shouldn't the last_update date be changed?